app/src/main/java/com/github/fribourgsdp/radio/data/StarterPlaylists.kt

```diff
@@ -0,0 +1,452 @@
+package com.github.fribourgsdp.radio.data
+
+import com.github.fribourgsdp.radio.data.Genre.*
+
+object StarterPlaylists {
```

Comment:
Why did you choose to write the playlist in hard copy and not in a file?

Reply:
Done !